packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx

```diff
@@ -154,8 +158,8 @@ export const TreeNodeRenderer: ForwardRefExoticComponent<TreeNodeRendererProps &
             onNodeKeyDown?.(node, !selected, event);
           }
         }}
-        icon={getIcon ? getIcon(node) : undefined}
         actions={getActions()}
+        unstable_decorations={getDecorations && [getDecorations.map((getDecoration) => getDecoration(node))]}
```

Comment:
How does this work? As far as I can tell, `unstable_decorations` should be a `React.ReactNode`, but we're passing `React.ReactNode[][]`.

Reply:
Removed left over [] from testing. It should now return `ReactElement[]`, don't really want to allow `ReactNode` as it includes too much unnecessary things.